src/helpers/package.js

```diff
@@ -15,7 +16,21 @@ export const hasPackageJson = () => {
 
 export const ensurePackageJson = () => {
   if (!hasPackageJson()) {
-    stopWithError(`No ${PACKAGE} in current directory`)
+    console.log(chalk.red(`No ${PACKAGE} in current directory`))
+    const value = askValue(
+      chalk.yellow('Do you use to create one for you? [y/N]: '),
+      ['y', 'n', 'yes', 'no', '']
+    )
+    if (['y', 'yes', ''].includes(value)) {
```

Comment:
If the default value is yes then put a capital `Y` and lowercase `n` in the prompt